carsus/io/cmfgen/base.py

```diff
@@ -455,27 +455,45 @@ class CMFGENReader:
     ----------
     levels : DataFrame
     lines : DataFrame
+    collisions : DataFrame
 
     """
 
-    def __init__(self, data, priority=10):
+    def __init__(self, data, collisions=False, priority=10, temperature_grid=None):
         """
         Parameters
         ----------
         data : dict
             Dictionary containing one dictionary per species with 
             keys `levels` and `lines`.
-
```

Comment:
Missing definition of `collisions`

Reply:
Done in 4dac9194e1390d04440482ad75947bb8c44143a5